linked_list/linked_list.py

```diff
@@ -1,75 +1,138 @@
-
 # Defines a node in the singly linked list
 class Node:
 
-    def __init__(self, value, next_node = None):
+    def __init__(self, value, next_Node = None):

```

Comment:
With Python Pep8 style conventions, we'd like our variables to be all lowercase. You also use previous in your methods below, but no attribute previous exists.
```suggestion
    def __init__(self, value, next_node = None, previous_node = None):

```